Fix octavation and typo in two BWV files.

hjunes opened the pull request:
Fix octavation and typo in two BWV files. Even if compiling was tested to be passed using LilyPond 2.19.25, the version number was not elevated.

dominicus commented:
updates confirmed....
@hjunes, thanks for taking the time to submit the correction. Please run convert.ly to bring to 2.18.2 (or later) and push the updated files.

hjunes commented:
Ok,

now updated.

Heikki

2015-08-21 9:08 GMT+03:00 Javier Ruiz-Alma <notifications@github.com>:

> updates confirmed....
> @hjunes <https://github.com/hjunes>, thanks for taking the time to submit
> the correction. Please run convert.ly to bring to 2.18.2 (or later) and
> push the updated files.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MutopiaProject/MutopiaProject/pull/628#issuecomment-133298492>
> .
>

hjunes commented:
Ok, great!

I have fixed and updated another piece.

Br,
Heikki

2015-08-21 16:58 GMT+03:00 Chris Sawer <notifications@github.com>:

> Merged #628 <https://github.com/MutopiaProject/MutopiaProject/pull/628>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MutopiaProject/MutopiaProject/pull/628#event-388819410>
> .
>
